src/controller/user.py

```
@@ -315,3 +364,362 @@ async def reactivate_oldboy_ctrl(session: SessionDep, user: User) -> None:
+    ):
+        user = self.session.get(User, body.id)
+        if not user:
+            raise HTTPException(404, detail="user not found")
+        if user.status == UserStatus.active:
+            raise HTTPException(409, detail="the user is already active")
+        user.status = UserStatus.active
+        self.session.add(user)
+        standbyreq = self.session.exec(
+            select(StandbyReqTbl)
+            .where(StandbyReqTbl.standby_user_id == body.id)
+            .where(StandbyReqTbl.is_checked == False)
```

Ruff (0.14.3)

Avoid equality comparisons to `False`; use `not StandbyReqTbl.is_checked:` for false checks

Replace with `not StandbyReqTbl.is_checked`

(E712)